app/view/show.py

```diff
@@ -13,6 +13,32 @@
 from collections import defaultdict
 
 
+def add_bug_link(cves, pkgs, severity):
+    references = '\n'.join((cve.reference for cve in cves))
```

Comment:
as we maybe want to do something with the cve data itself, we have more advantage if we siply pass the cves array to the render_template function and iterate over it in the bug.txt template. This way we could later use the cves array to maybe populate some info in the description from the very same data we use to populate the references from